lib/takeaway.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'sms_notificaiton'
+require_relative 'menu'
+
+class Takeaway
+  attr_accessor :food
+  attr_reader :current_order, :order_total, :order_summary
+
+  def initialize(menu_klass = Menu.new, sms_notification_klass = SmsNotification.new)
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)